test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -8,9 +8,13 @@ function testMultiplyFractionsSearch(exprString, outputStr) {
 
 describe('multiplyFractions', function () {
```

Comment:
I disagree, I feel like it would be weird to call multiply in cancel code. It seems more intuitive to multiply and then cancel at the end. But I will rename the file :)